### DISCUSSION TITLE
TP-Link Smartplug Support

### DISCUSSION THREAD

**Discussion by fhb** [UnchartedBull/OctoDash#1809] [Ideas]
I would like to suggest support of "TP-Link Smartplug" plugin similar to how "PSU-Control" is used.

**Comment by jneilliii** [comment]
It is already in the latest Beta version of OctoDash, along with Tasmota and TasmotaMQTT plugin support.

**Reply by fhb** [reply]
Thank you for the fast response!

**Reply by ichibana** [reply]
Hi! Apologies if I'm missing something basic, but I don't understand how to add TP Link control to the homescreen for OctoDash. That's one of the main reasons I'm trying to add a touchscreen; I'd like to not have to get out my phone just to turn on my printer. I've toggled "tpLinkSmartPlug" to TRUE... but no change. Thanks in advance.

**Reply by ichibana** [reply]
I've added the @TPLINKOFF command to the custom icons option and it works! Great!
